ports/psoc6/modules/network/network_ifx_whd.c

```diff
@@ -34,38 +34,41 @@
 #if MICROPY_PY_NETWORK_IFX_WHD
 
 #include "lwip/netif.h"
+#include "lwip/dns.h"
 #include "whd.h"
 #include "whd_network_types.h"
 #include "whd_wlioctl.h"
 #include "network_ifx_whd.h"
 #include "whd_wifi_api.h"
 #include "extmod/modnetwork.h"
-#include "cy_wcm.h"
+#include "cy_wcm.c"
```

Comment:
I will bet this inclusion of .c is due to some static or private scope variables or objects of the cy_wcm.c?